packages/security_center/lib/disk_encryption/disk_encryption_page.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:security_center/disk_encryption/disk_encryption_providers.dart';
+import 'package:security_center/l10n/app_localizations.dart';
+import 'package:security_center/services/disk_encryption_service.dart';
+import 'package:security_center/widgets/scrollable_page.dart';
+import 'package:yaru/yaru.dart';
+
+class DiskEncryptionPage extends ConsumerWidget {
+  const DiskEncryptionPage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return _Body();
+  }
+}
+
+class _Body extends StatelessWidget {
+  const _Body();
+
+  @override
+  Widget build(BuildContext context) {
+    final l10n = AppLocalizations.of(context);
+    return ScrollablePage(
+      children: [
+        Text(
+          l10n.diskEncryptionPageRecoveryKey,
+          style: Theme.of(context).textTheme.titleMedium,
+        ),
+        const SizedBox(height: 16),
+        Text(l10n.diskEncryptionPageStoreYourKey),
+        const SizedBox(height: 16),
```

Comment:
There's a convenient `separatedBy` method in the `iterable_extensions.dart` here :)